Strategies: Use new API

niweb opened the pull request:
In this PR the new API and the new data structure were implemented for everything related to strategies:
* No more mock data
* New entity hierarchy: Strategy -> Block -> Category -> Situation -> Measure

Additionally:
* Removed all `export * from ...` and replaced by named exports for better IDE support
* API Response objects get automatically converted from snake_case keys to camelCase keys
* Added typings for the data structure the API returns and explicitly convert to client side data format
* Define Route Parameter names in App Core so they can be used with `useParams` by any pages

MaxF12 commented:
This seems to break the display of the country flags on the analysis page (not sure why)

niweb commented:
Ah yes. I changed the client api, so that all object keys of the responses are in camelCase. This of course broke the snake_case flag in Analysis. I fixed it.
